observer/lib/observer.js

```diff
@@ -1,25 +1,26 @@
 import { updateDailyTransferStats } from './platform-stats.js'
+import * as Sentry from '@sentry/node'
 
 /**
- * @param {import('pg').Pool} pgPool
+ * @param {import('@filecoin-station/spark-stats-db').pgPools} pgPools
  * @param {import('ethers').Contract} ieContract
  * @param {import('ethers').Provider} provider
  */
-export const observe = async (pgPool, ieContract, provider) => {
+export const observe = async (pgPools, ieContract, provider) => {
   await Promise.all([
-    observeTransferEvents(pgPool, ieContract, provider),
-    observeScheduledRewards(pgPool, ieContract)
+    observeTransferEvents(pgPools.stats, ieContract, provider),
+    observeScheduledRewards(pgPools, ieContract)
   ])
```

Comment:
What is the purpose of this `observe` method when the file `observer/bin/spark-observer.js` has its own logic to compose these two observers in a different way than presented here? Can we either move that code from `observer/bin/spark-observer.js` to this file, or else remove the `observe()` method entirely?

Reply:
It's for `dry-run`, so that this file can run all observe functions without needing to know all. Or would you rather inline this logic there?

Reply:
move to dry run